Python/compile.c

```diff
@@ -4764,6 +4818,8 @@ compiler_async_comprehension_generator(struct compiler *c,
     compiler_pop_fblock(c, ASYNC_COMPREHENSION_GENERATOR, start);
 
     compiler_use_next_block(c, except);
+    c->u->u_lineno = -1;
```

Comment:
Likewise, is this due to the other changes or something you happened to notice?

Reply:
I've reverted this change. Handling of re-raising exceptions needs to be fixed for both 3.10 and 3.11, but will need different treatments. See https://bugs.python.org/issue43933